docs/pages/deep-sleep-manager_L3_TestProcedure.md

```diff
@@ -201,20 +216,20 @@ All tested wake-up sources must successfully trigger the system to exit deep sle
 
     The test will set the wake-up source and trigger a deepsleep.
 
-    The above will loop through all supported wake up sources that applicable for the specific device based on the .yaml file.
+    The above will loop through all supported wake up sources that applicable for the specific device based on the `yaml` file.
 
-- Once in deepsleep the device will need to be manually awoken for some sources like IR, CEC and Power_Key.
+- Once in deepsleep the device will need to be manually awoken for some sources like `IR`, `CEC` and `Power_Key`.
 
-    - When the device enters 'IR' test, to trigger the wake up use an IR remote pressing the power key on the remote to wake up the device. If the device successfully wakes up from deepsleep via the specified method, User should enter the response it passes the test.
+    - During the `IR` test, use an `IR` remote to trigger the wake-up by pressing the power key. If the device successfully wakes up from deep sleep using this method, the test evaluates whether the wake-up process was successful.
 
-    - When the device enters 'power key' test, to trigger the wake up press the front panel power key button of the device itself. If the device successfully wakes up from deepsleep via the specified method, User should enter the response it passes the test.
+    - During the `Power_Key` test, press the power key of the `dut` to wake-up. If the device successfully wakes up from deep sleep using this method, the test evaluates whether the wake-up process was successful.
```

Comment:
Porbably the IR wakeup keys can be Power and Home Keys.

Reply:
updated the line with home key